subsys/net/lib/sockets/sockets_service.c

```diff
@@ -277,15 +277,23 @@ static void socket_service_thread(void)
 	k_condvar_broadcast(&wait_start);
 }
 
-K_THREAD_DEFINE(socket_service_monitor, CONFIG_NET_SOCKETS_SERVICE_STACK_SIZE,
-		socket_service_thread, NULL, NULL, NULL,
-		CLAMP(CONFIG_NET_SOCKETS_SERVICE_THREAD_PRIO,
-		      K_HIGHEST_APPLICATION_THREAD_PRIO,
-		      K_LOWEST_APPLICATION_THREAD_PRIO), 0, 0);
-
 static int init_socket_service(void)
 {
-	k_thread_name_set(socket_service_monitor, "net_socket_service");
+	static k_tid_t ssm;
```

Comment:
Should `ssm`this be `static`? We only use thread ID to set the name, I don't think there's a need to keep that memory after.

Reply:
Right, not need for static. Will fix.